Tasks/PublishSymbols/Helpers.ps1

```diff
@@ -142,9 +104,10 @@ function Add-SourceServerStream {
             [string[]]$pdbStrArgs = @(
                 '-w'
                 "-p:""$SymbolsFilePath"""
-                "-i:""$streamContentFilePath"""
+                "-i:""$($streamContentFile.FullName)"""
                 '-s:srcsrv'
             )
+            $OFS = " "
```

Comment:
This is just good practice to set $OFS before you use it. It's set within the scope of the function, so we don't need to worry about it causing issues upstream.